fasthttpadaptor/request.go

```diff
@@ -2,31 +2,31 @@ package fasthttpadaptor
 
 import (
 	"bytes"
+	"github.com/valyala/fasthttp"
 	"io/ioutil"
 	"net/http"
 	"net/url"
-
-	"github.com/valyala/fasthttp"
+	"unsafe"
 )
 
 // ConvertRequest convert a fasthttp.Request to an http.Request
 // forServer should be set to true when the http.Request is going to passed to a http.Handler.
```

Comment:
```suggestion
// ConvertRequest convert a fasthttp.Request to an http.Request
// forServer should be set to true when the http.Request is going to passed to a http.Handler.
//
// The http.Request must not be used after the fasthttp handler has returned!
// Memory in use by the http.Request will be reused after your handler has returned!
```

This was already unsafe before your change because of the use of `ctx.PostBody()`. But with your change it becomes even more dangerous so it's best if we clearly document this.